src/preload/script.ts

```diff
@@ -40,6 +40,18 @@ function getCalcDesktopURL() {
 async function load(pluginsForceDisabled: Set<string>) {
   if (window.location.href.includes("geometry-legacy")) return;
 
+  Console.warn(
+    BROWSER === "firefox"
+      ? `%cThe warning above (Loading failed for the <script> with source...) is intentional and does not indicate a bug.`
+      : `%cThe error above (GET https://... net::ERR_BLOCKED_BY_CLIENT) is intentional and does not indicate a bug.`,
```

Comment:
```suggestion
      : `%cThe error above (net::ERR_BLOCKED_BY_CLIENT) is intentional and does not indicate a bug.`,
```
Since it doesn't match for me currently.
![image](https://github.com/DesModder/DesModder/assets/20214911/1b2ee208-24dc-4c2f-9f1e-607f8cd95723)